packages/syft/proto/core/tensor/fixed_precision_tensor.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package syft.core.tensor;
+
+import "proto/core/tensor/tensor.proto";
+import "proto/core/tensor/share_tensor.proto";
+import "proto/lib/numpy/array.proto";
+
+message FixedPrecisionTensor {
+    oneof data {
+        Tensor tensor = 1;
+        ShareTensor share = 2;
```

Comment:
Q: Why do we have ShareTensor inside FP Tensor.Should it be like FP tensor inside ShareTensor.

Reply:
Here we have FPTensor --> ShareTensor  (you need to convert to integer and the we secret share it -- and the result is still an int tensor)